pizzeria/styles/_gallery.scss

```diff
@@ -0,0 +1,41 @@
+body {
+  display: flex;
```

Comment:
don't write such general css in gallery file

and are you aware that this will apply for body on all pages, not only for gallery?